frontend/src/app/shared/components/product-item/product-item.component.scss

```diff
@@ -0,0 +1,76 @@
+@import 'colors';
+@import 'variables';
+@import 'mixins';
+
+.best-product-item {
+  border: 1px solid $color-grey-050;
+  padding: 16px;
+  width: 270px;
+  border-radius: 12px;
```

Comment:
Sa pui proprietatile intr-o ordine, o sa folosim mecanismul asta 

https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/